### PR TITLE
Feature/bio conductor docs

### PR DESCRIPTION
**Pull request by paselem**
Examples on how to use BioConductor in doAzureParallel

### CODE REVIEW

File: docs/20-package-management.md [hunk 1]
```diff
@@ -70,6 +70,32 @@ results <- foreach(i = 1:number_of_iterations, .packages=c('package_1', 'package
...
 Installing packages from github using this method is not yet supported.
 
+## Installing Packages from BioConductor
+Currently there is no native support for Bioconductor package installation, but it can be achieved by installing the packages directly in your environment or using the 'commandLine' feature in the cluster configuration. We recommend using the 'commandLine' to install the base BioConductor package and then install additional packages either through the 'commandLine' as well, or directly in your code.
+
+### Installing BioConductor using the 'commandLine'
+
+We recommend using the [script provided in the samples](../samples/package_management/bioc_setup.sh) section of this project which will install the required pre-requisites for BioConductor as well as BioConductor itself.
+
+Simply update your cluster configuration commandLine as follows:
```
Comment (brnleehng) [inline docs/20-package-management.md]: Should we provide a pre built cluster configuration file?

Comment (paselem) [reply docs/20-package-management.md]: I do that in the samples. Do you think we need to have one in the docs as well? I can point to it from the docs if you think that will help.

Comment (paselem) [reply docs/20-package-management.md]: Ok... I see the issue. The .gitignore file is ignoring all .json files so it did not show up. I updated it and pushed.

Comment (brnleehng) [reply docs/20-package-management.md]: LGTM

**Pull request merged**